### ISSUE TITLE
PostgreSQL Explain function does not respect anonymous parameter setting

### ISSUE THREAD

**Issue by y6nH** [dbeaver/dbeaver#3145]
In Global Settings, under "SQL Processing", disable "Anonymous SQL parameters", or set the character used to something other than "?".

This allows you to run Postgres queries using the ? operator. For example, if the "person" table has a "prefs" JSONB column and you want to select the people with a language preference:
```
select id
  from person
  where prefs ? 'language'
```
With default settings, running this query prompts for a parameter value. With anonymous parameters off, it works fine.

Now press the Explain button, or select it from the SQL Editor menu. Despite anonymous parameters being disabled, the operation fails with the message `SQL Error [22023]: No value specified for parameter 1.`

The workaround is to type EXPLAIN in before the query manually, but it would be nice if the button worked too.

**Comment by offstate** [comment]
Also Explain function doesn't parse parameters in a query.
This query work fine when I run it.
```
select 1
where 1=:f
```

But when I try to get execution plan I receive an error
**SQL Error [42601]: ERROR: syntax error at or near ":"**

**Comment by Kireyko** [comment]
Hi all,
this issue is still actual and was not fixed in last version of dbeaver(5.2.3).
Is it planned to be fixed in near future?

**Comment by uslss** [comment]
already fixed in 7.0
